src/composables/pools/usePoolCreation.ts

```diff
@@ -92,8 +92,12 @@ export default function usePoolCreation() {
         w => w.tokenAddress
       );
 
-      poolCreationState.name = getPoolSymbol();
-      poolCreationState.symbol = getPoolSymbol();
+      if (!poolCreationState.name) {
+        poolCreationState.name = getPoolSymbol();
+      }
```

Comment:
Is it possible to clean this up a bit with?
```suggestion
      !poolCreationState.name && poolCreationState.name = getPoolSymbol();
```
Same for conditional below if it works.

Reply:
Could also be `poolCreationState.name = poolCreationState.name || getPoolSymbol()`